XF.Material/UI/MaterialTextField.xaml.cs

```diff
@@ -193,10 +193,10 @@ public bool AlwaysShowUnderline
         /// <summary>
         /// Gets or sets the background color of this text field.
         /// </summary>
-        public new Color BackgroundColor
+        public Color RealBackgroundColor
```

Comment:
Not sure if i like the name RealBackgroundColor... Do you have better ideas instead of `Real`?